lib/devengo/resources/members/products/collection.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Devengo
+  module Resources
+    module Members
+      module Products
+        class Collection < Shared::BaseCollection
```

Comment:
We agreed to [no longer use the Shared Collection](https://github.com/devengoapp/core/issues/4425). This applies here, doesn't it?

Reply:
No, this convention would not apply in this case